README.md

```diff
@@ -1,106 +1,55 @@
-</p>
-<p align="middle">A Guides component that can draw ruler and manage guidelines.</p>
-<p align="middle">
-    <a href="https://daybrush.com/guides" target="_blank"><strong>Demo</strong></a> /
-    <a href="https://daybrush.com/guides/release/latest/doc/" target="_blank"><strong>API</strong></a> /
-    <a href="https://github.com/daybrush/ruler" target="_blank"><strong>Ruler</strong></a> /
-    <a href="https://github.com/daybrush/scena" target="_blank"><strong>Main Project</strong></a>
-</p>
+1. Run `npm run build` in the folder *packages/react-guides*;
+2. Run `npm run build` in root of the library 'Guides';
+3. Check the *dist/guides.js*, if the changes updates; 
+3. Install the changed library *react-guides* from *[packages/react-guides](https://github.com/ProtoPie/guides/tree/master/packages/react-guides)* to the *[Guides](https://github.com/ProtoPie/guides/blob/master/package.json)*
 
 
-## ⚙️ Installation
-### npm
-```sh
-$ npm i @scena/guides
-```
-
-### scripts
-```html
-<script src="//daybrush.com/guides/release/latest/dist/guides.min.js"></script>
-```
-
-## 🚀 How to use
-```tsx
-
-import Guides from "@scena/guides";
+### Guides
+## :wastebasket: Delete a guide
 
-const guides = new Guides(document.body, {
-    type: "horizontal",
-}).on("changeGuides", e => {
-    console.log(e.guides);
-});
+**We have two ways how to delete a guide:**
+1. Select a line and press the button 'Delete';
+2. Click on a line and drag to the relative ruler.
+# Functions
 
+- `deleteSelectedGuide` - this function is used when a user selected a line and press the 'Delete' button;
 
-let scrollX = 0;
-let scrollY = 0;
-window.addEventListener("resize", () => {
-    guides.resize();
-});
-
-window.addEventListener("wheel", e => {
-    scrollX += e.deltaX;
-    scrollY += e.deltaY;
-
-    guides.scrollGuides(scrollY);
-    guides.scroll(scrollX);
-});
-
+```tsx
+    public deleteSelectedGuide() {
+        // get all active guides
+        const guides = this.getGuides();
+        // find the index of the guide which the user selected
+        const index = guides.findIndex(guide => {
+            if(this.state.selectedGuides.includes(guide)){
+                return guide;
+            }
+        });
+        // call the event what return the index and value of a deleted guide
+        /**
+         * When the drag is deleted, the deleteGuide event is called.
+         * @memberof Guides
+         * @event deleteGuide
+         * @param {OnDeleteGuide} - Parameters for the deleteGuide event
+         */
+        this.props.onDeleteGuide!({
+            deletedPosGuide: guides[index],
+            deletedIndexGuide: index,
+        });
+        
+        guides.splice(index, 1);
+        // push new arrays of guides to the state and clear the array which contains a selected line
+        this.setState({
+            guides,
+            selectedGuides: []
+        });
+    }
```

Comment:
I'm not sure if we should spend time documenting this in such a way.
This is not a public package.
I believe using TS in right way is the best documentation.
Also, using comments inside code is also a good way.
Another good solution is to create auto documentation from https://github.com/TypeStrong/typedoc

Reply:
But I'm not sure if we want spent time documentation this project, but anyway I would like to know what team things about it